server/src/api/roster/roster.controller.ts

```diff
@@ -326,9 +326,17 @@ class RosterController {
     res.json(entry);
   }
 
+  async savePhoneNumber(edipi: string, phoneNumber: string) {
```

Comment:
Can we move this function out of the controller class? Our convention has been to only have endpoints in the controller itself. Typically, we'd put this sort of function in `roster-utils.ts` since it's used outside of this controller.

The function name may need to be made more explicit in that case, since the functions aren't wrapped in a class/namespace (which I'd like to fix at some point). So maybe something like `saveRosterPhoneNumber` would be good.

Reply:
fixed